tests/cli.nix

```diff
@@ -8,7 +8,8 @@ runCommand "test-cli"
   {
     "kind": "Secret",
     "data": {
-      "foo": "bar"
+      "foo": "bar",
+      "bar": {}
```

Comment:
To make this an example of #374, you'll need to add `--json bar {}` below.
Re-using "bar" here is a bit confusing though. Another placeholder, like `baz` would be clearer.

Reply:
please review it again.